File: docs/test_analytics.py
from analytics import log_feature_usage

# Test data
user_id = "test_user_001"
video_id = "test_vid_001"
feature = "TTS"
metadata = {"language": "en", "duration": "15s"}

# Run test
result = log_feature_usage(user_id, video_id, feature, metadata)
print("Analytics logged:", result)

File: docs/test_analytics.js
import { logFeatureUsage } from './analytics.js'; // Adjust path if needed

// Test data
const userId = "test_user_001";
const videoId = "test_vid_001";
const action = "TTS";
const metadata = {
  language: "en",
  duration: "15s"
};

// Run test
logFeatureUsage(userId, videoId, action, metadata)
  .then(result => {
    console.log("Analytics logged:", result);
  })
  .catch(error => {
    console.error("Error logging analytics:", error);
  });
